docs/next.config.js

```diff
@@ -136,22 +136,22 @@ module.exports = withNextPluginPreval({
       },
       /**
        * source: a url has one of the folder's names (components, getting-started, guides, theming)
-       * destination: add '[platform]' to the the beginning
+       * destination: add '/react/' to the the beginning
```

Comment:
since we only allow the platform names to pass the validation, `[platform]` doesn't work anymore.